### PR TITLE
Updated screenshots

### LINKED ISSUE #5054: Documentation — Missing rapid7-pf.png for installation guide
The file `docs/images/rapid7/rapid7-pf.png` is missing.

### LINKED ISSUE #5055: Documentation — update screenshots to v9 GUI
There are still many screenshots to update to reflect the new Web admin GUI from version v9.

Examples:

* mobile-iron-pf-config.png
* rapid7/rapid7-node-tab.png
* doc-paloalto-cfg_SSO_pf.png
* doc-jsonrpc-cfg_SSO_pf.png
* doc-fortigate-cfg_SSO_pf.png
* doc-checkpoint-cfg_SSO_pf.png
* etc

### PR DESCRIPTION
**Pull request by Ieno77**
# Description
Updated screenshots 

# Impacts
Documentations

# Delete branch after merge
(REQUIRED)
YES

# Issue
fixes #5054
fixes #5055

### CODE REVIEW

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 1]
```diff
@@ -3758,11 +3758,7 @@ Notes on the configuration:
 * Scan before/on/after registration controls when the automated scans are started for the devices PacketFence sees. If you only want to start the scans manually, leave those unchecked.
 * You will not be able to select a scan template, site and scan engine when initially configuring the engine. First configure the access and credentials and edit the engine again to be able to select those from the available values in Rapid7.
 
-image::rapid7/rapid7-pf-1.png[scaledwidth="100%",alt="Rapid7 PacketFence config"]
-
-image::rapid7/rapid7-pf-2.png[scaledwidth="100%",alt="Rapid7 PacketFence config"]
-
-image::rapid7/rapid7-pf-3.png[scaledwidth="100%",alt="Rapid7 PacketFence config"]
+image::rapid7/rapid7-pf.png[scaledwidth="100%",alt="Rapid7 PacketFence config"]
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: Remove old rapid7-pf-X files if they are not used anymore.

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 2]
```diff
@@ -4123,9 +4119,7 @@ Next, go in the 'NTLM cache' tab and:
  * Adjust the 'Expiration'
  * Enable 'NTLM cache background job' and/or 'NTLM cache on connection'. In the case of this example, both will be enabled.
 
-image::domain-ntlm-cache-1.png[scaledwidth="100%",alt="Domain NTLM cache"]
-
-image::domain-ntlm-cache-2.png[scaledwidth="100%",alt="Domain NTLM cache"]
+image::domain-ntlm-cache.png[scaledwidth="100%",alt="Domain NTLM cache"]
```
Comment (nqb) [inline docs/PacketFence_Installation_Guide.asciidoc]: - Remove old domain-ntlm-cache-X files if they are not used anymore.
- GitHub report a conflict on domain-ntlm-cache.png that prevent a merge

File: docs/pki/microsoft.asciidoc [hunk 3]
```diff
@@ -229,15 +229,13 @@ Add a new PKI provider and select SCEP.
...
 
-image::scep-ms-pki-provider.png[scaledwidth="100%",alt="MSPKI configuration"]
-
 For the URL it will be `http://<ServerDNSName>/CertSrv/mscep/`.
 
 WARNING: Don't use `https:` scheme.
 
 You do not need any Username/Password combination for this configuration.
 
-image::scep-ms-pki-provider2.png[scaledwidth="100%",alt="MSPKI configuration"]
```
Comment (nqb) [inline docs/pki/microsoft.asciidoc:240]: Remove this file if not needed anymore.

File: docs/pki/packetfence.asciidoc [hunk 4]
```diff
@@ -321,8 +321,6 @@ Pay attention to the username and password which have to match an authorized use
 
 image::packetfence-pki-provider.png[scaledwidth="100%",alt="PacketFence PKI configuration"]
 
-image::packetfence-pki-provider2.png[scaledwidth="100%",alt="PacketFence PKI configuration"]
```
Comment (nqb) [inline docs/pki/packetfence.asciidoc]: Remove this file if not needed anymore.

**Review by nqb — CHANGES_REQUESTED** [review]

**Comment by nqb** [comment]
Hello @Ieno77, 

Could you update description of this PR with links to issues related to images (like #5055 and #5054) ?

Thanks

**Comment by nqb** [comment]
@Ieno77, you certainly need to rebase this PR.

File: docs/PacketFence_Installation_Guide.asciidoc (hunk 1 above)
Comment (Ieno77) [reply docs/PacketFence_Installation_Guide.asciidoc]: done

**Comment by julsemaan** [comment]
@nqb, can you review again

**Comment by cgx** [comment]
Don't forget to [crunch](https://github.com/chrissimpkins/Crunch) your images.

**Comment by nqb** [comment]
Need to be rebased after merge of #5170. 

leno77, could you crunch your images ?

**Comment by cgx** [comment]
@Ieno77 Good! Now it's time to squash your commits.

**Comment by nqb** [comment]
@Ieno77, could you rebase ?

**Review by nqb — CHANGES_REQUESTED** [review]
- docs/images/aerohive-pf-config-webauth.png is not correct (no config displayed)
- docs/images/aruba-registration-acl.png: doesn't show something related to registration
- files that should have been deleted are still present (see previous comments)
- docs/images/dpsk_provisioner.png show a Windows provisioner
- rebase for new pf-pki images
- docs/images/pfaerohive09.png related related to aerohive ?
- docs/images/pfaerohive13.png related to aerohive ?

**Comment by Ieno77** [comment]
Please @nqb can you review and merge ?

**Comment by cgx** [comment]
@Ieno77 Please use lowercase letters for all filenames.

**Review by nqb — CHANGES_REQUESTED** [review]
- aerohive-pf* are not used anywhere
- previous remark still true for dpsk_provisionner.png

**Comment by Ieno77** [comment]
@nqb I removed useless screens and changed the aruba-registration-acl
For the dpsk_provisionner, the old screen was already showing a Windows provisionner

**Comment by nqb** [comment]
>  For the dpsk_provisionner, the old screen was already showing a Windows provisionner

It makes sense regarding location of this screen in documentation.

**Pull request merged**